tests/system/node/3/config.yaml

```diff
@@ -0,0 +1,63 @@
+
+################################################################################
+##                         Ban manager configuration                          ##
+################################################################################
+banman:
+  max_failed_requests: 10
+  ban_duration: 86400
+
+################################################################################
+##                          Administrative interface                          ##
+################################################################################
+admin:
+  enabled: false
+  address: 127.0.0.1
+  port:    2827
+
+################################################################################
+##                               Node discovery                               ##
+##                                                                            ##
+## When the network first starts, we need to connect to some peers to learn   ##
+## the topology and find a safe intersection to listen to, and, if we are     ##
+## a validator, to insert ourselves.                                          ##
+################################################################################
+network:
+  # Supported value: IPv4, IPv6
+  - http://node-0:2826
+  - http://node-1:3826
+  - http://node-2:4826
+
+################################################################################
+##                               Quorum slices                                ##
+################################################################################
+quorum:
+  threshold: 66%
+  nodes:
+    - GBUVRIIBMHKC4PE6BK7MO2O26U2NJLW4WGGWKLAVLAA2DLFZTBHHKOEK
+
```

Comment:
That should be removed as well

Reply:
This will cause failures because the config parser always tries to parse quorum slices.

There is another issue about removing the quorum config from the config though.

Reply:
If it remove this, Clearly, a quorum configuration analysis error occurs.